instrumentation/jedis/jedis-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v1_4/JedisDbAttributesGetter.java

```diff
@@ -7,13 +7,15 @@
 
 import io.opentelemetry.instrumentation.api.incubator.semconv.db.DbClientAttributesGetter;
 import io.opentelemetry.instrumentation.api.incubator.semconv.db.RedisCommandSanitizer;
-import io.opentelemetry.javaagent.bootstrap.internal.AgentCommonConfig;
 import io.opentelemetry.semconv.incubating.DbIncubatingAttributes;
 
 final class JedisDbAttributesGetter implements DbClientAttributesGetter<JedisRequest, Void> {
 
-  private static final RedisCommandSanitizer sanitizer =
-      RedisCommandSanitizer.create(AgentCommonConfig.get().isStatementSanitizationEnabled());
+  private final RedisCommandSanitizer sanitizer;
+
+  JedisDbAttributesGetter(boolean statementSanitizerEnabled) {
+    this.sanitizer = RedisCommandSanitizer.create(statementSanitizerEnabled);
+  }
```

Comment:
removed dependency on GlobalOpenTelemetry from Getters/Extractors (for a future world where they're used by library instrumentation)